addon/components/hyper-table/filters-renderers/date.js

```diff
@@ -26,7 +26,11 @@ export default Component.extend({
 
   movingDateOptions: {
     'Today': 'today',
-    'Yesterday': 'yesterday'
+    'Yesterday': 'yesterday',
+    'This Week': 'This Week',
+    'Last Week': 'Last Week',
+    'This Month': 'This Month',
+    'This Year': 'This Year',
```

Comment:
can you put the values in lowercase + underscore as they are used as "ids" and don't have to be humanized